test-app/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "test-app",
+  "version": "2.0.6",
+  "description": "The simple DDAU component built for Ember",
+  "keywords": [],
+  "repository": "https://github.com/ember-a11y/ember-select-light",
+  "license": "MIT",
+  "author": "Ava Wroten <ava@wroten.me> (https://www.wroten.me/)",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.2.0",
+    "@glimmer/component": "^1.0.3",
+    "@glimmer/tracking": "^1.0.3",
+    "@kb/ember-select-light": "2.0.6",
+    "babel-eslint": "^10.1.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^1.10.1",
+    "ember-cli": "~3.25.3",
+    "ember-cli-babel": "^7.23.1",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-htmlbars": "^5.3.2",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-terser": "^4.0.1",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-export-application-global": "^2.0.1",
+    "ember-load-initializers": "^2.1.2",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-page-title": "^6.2.1",
+    "ember-qunit": "^5.1.2",
+    "ember-resolver": "^8.0.2",
+    "ember-source": "~3.25.1",
```

Comment:
We should probably cross 4.0 here + drop support for Node < 16. This _is_ a major bump.